pypeit/spectrographs/keck_nirspec_high.py

```diff
@@ -0,0 +1,1434 @@
+# Code to make the NIRSPEC DRP
```

Comment:
I think we should consolidate `keck_nirspec.py` and `keck_nirspec_high.py`.  Or, at least, we should only have one `KeckNIRSPECSpectrograph` class.  There is now one defined by both of these files.  This isn't technically a problem because they're defined in different modules, but it's not very clear.